components/MemberCard.tsx

```diff
@@ -0,0 +1,36 @@
+import { StaticImageData } from "next/image";
+import Image from "next/image";
+import { useState } from "react";
+
+const MemberCard = ({imageSrc, firstName, lastName, description, position}: MemberCardProps) => {
+    const [hovered, setHovered] = useState(false);
+
+    return(
+        <div className="flex flex-col items-center justify-center rounded bg-wato-grey-porcelain lg:w-[8vw] lg:min-w-[8vw]"
+            onMouseEnter={() => {setHovered(true)}}
+            onMouseLeave={() => {setHovered(false)}}
+        >
+            <Image alt="Image of member" src={imageSrc} className="rounded-full p-3" />
+            <h2 className="font-semibold">{firstName + ","}</h2>
+            <h2 className="pb-1 font-semibold">{lastName}</h2>
+            <h4 className="pb-1 font-normal">{position}</h4>
```

Comment:
I don't have semibold mapped to anything. Refer to `Layout.txt` to see the font weights I've mapped.
Let's also add some negative padding, and change the color of "position" as per the figma
```jsx
<h2 className="font-black -mb-2">{firstName}</h2>
<h2 className="font-black">{lastName}</h2>
<h4 className="text-wato-blue">{position}</h4>  
```